src/datachain/lib/file.py

```diff
@@ -447,13 +451,60 @@ class ImageFile(File):
     def read(self):
         """Returns `PIL.Image.Image` object."""
         fobj = super().read()
-        return Image.open(BytesIO(fobj))
+        return PilImage.open(BytesIO(fobj))
 
     def save(self, destination: str):
         """Writes it's content to destination"""
         self.read().save(destination)
 
 
+class Image(DataModel):
+    """`DataModel` for image file meta information."""
+
+    width: int = Field(default=0)
+    height: int = Field(default=0)
+    format: str = Field(default="")
+
+
+class VideoFile(File):
+    """`DataModel` for reading video files."""
+
+
+class VideoClip(VideoFile):
+    """`DataModel` for reading video clips."""
+
+    start: float = Field(default=0)
+    end: float = Field(default=0)
+
+
+class VideoFrame(VideoFile):
+    """`DataModel` for reading video frames."""
+
+    frame: int = Field(default=0)
+    timestamp: float = Field(default=0)
+
+
+class Video(DataModel):
```

Comment:
Should it be a subclass of VideoFile?

Reply:
Good question. This is video meta information only. Do you think it will be used with `VideoFile` model only? What is the best name for this model then? 🤔